modules/ossm-uguide-routing-ingress.adoc

```diff
@@ -0,0 +1,228 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_user_guide/ossm-traffic-manage.adoc
+
+[id="ossm-routing-ingress_{context}"]
+= Managing ingress traffic
+
+In a service mesh, you can use a {ProductName} gateway to enable {ProductShortName} features such as monitoring and route rules to be applied to traffic entering the cluster.
+
+You can configure {ProductShortName} to expose a service outside of the service mesh using an {ProductShortName} gateway.
+
+.Before you begin
```

Comment:
I don't think we need this `Before you begin` section. We can (1) assume OSSM is installed and (2) user doesn't need to change to istio directory.